packages/web-components/fast-ssr/test/example.spec.ts

```diff
@@ -0,0 +1,10 @@
+import { test, expect, ElementHandle } from '@playwright/test';
+import fastSSR from "../src";
+
+test("example module test", async () => {
+	expect(fastSSR).toBe("fast-ssr");
+})
```

Comment:
I believe this whole test directory was removed, is this an artifact of the rebase?

Reply:
Oops. Probably. It was a very long rebase. Is it something you can add a commit to bring it back easily?

Reply:
Removed this folder.